packages/elastic-analytics/BUILD.bazel

```diff
@@ -36,6 +36,7 @@ NPM_MODULE_EXTRA_FILES = [
 #    "@npm//name-of-package"
 #    eg. "@npm//lodash"
 RUNTIME_DEPS = [
+  "@npm//moment",
```

Comment:
+1 for removing moment from the package and using plain old unix timestamps or Date objects

Reply:
we need `moment` to validate the ISO string. We could replace it with a regex, but I worry we might leave out some formats depending on the TZ.